tests/prompt/test_template.py

```diff
@@ -0,0 +1,202 @@
+# Copyright (c) 2022 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import unittest
+
+import numpy as np
+import paddle
+from parameterized import parameterized
+
+from paddlenlp.prompt import AutoTemplate, ManualTemplate, PrefixTemplate, SoftTemplate
+from paddlenlp.transformers import AutoModelForMaskedLM, AutoTokenizer
+
+
+class TestTemplate(unittest.TestCase):
```

Comment:
The class name of TestCase follows the rule: `{name}Test`, you can refer to: [transformers unittest](https://github.com/PaddlePaddle/PaddleNLP/blob/develop/tests/transformers/roberta/test_modeling.py#L399).

Reply:
Renamed as TemplateTest in https://github.com/PaddlePaddle/PaddleNLP/pull/4028/commits/8907013ca11532a3b5ba75c45ae34a59d70244a4